exporter/README.md

```diff
@@ -37,13 +37,48 @@ exporters:
 ```
 
 ## <a name="logging"></a>Logging
-TODO: document settings 
+TODO: document settings
 
 ## <a name="opencensus"></a>OpenCensus
-TODO: document settings 
+Exports traces and/or metrics to another OTel-Svc endpoint via gRPC.
+
+### <a name="opencensus-configuration"></a>Configuration
+
+* `endpoint`: target to which the exporter is going to send traces or metrics,
```

Comment:
Can this documentation move to the code base of [config.go](https://github.com/open-telemetry/opentelemetry-service/blob/master/exporter/opencensusexporter/config.go) to simplify keeping things up to date? 

Maybe we can say this is how a default configuration is enabled or what are the bare minimum of settings needed to get the exporter working and then refer to the test config.yaml for all future examples?

Reply:
Added to config.go, though I think it's still useful to also document the options in readme. I marked required vs. optional options.